### PR TITLE
Cleaning

### PR DESCRIPTION
**Pull request by jburel**
not much to test.
cc @pwalczysko

### CODE REVIEW

**Comment by joshmoore** [comment]
Re-opening to see if rtd will trigger a build.

**Comment by joshmoore** [comment]
Re-opening again, after attaching my GH account to RTD.

**Comment by joshmoore** [comment]
Re-opening after disabling PRs and re-enabling them.

**Comment by snoopycrimecop** [comment]
Conflicting PR. Removed from build [OMERO-guides-push#117](https://latest-ci.openmicroscopy.org/jenkins/job/OMERO-guides-push/117/). See the [console output](https://latest-ci.openmicroscopy.org/jenkins/job/OMERO-guides-push/117/consoleText) for more details.
Possible conflicts:
  - Upstream changes
    - README.md

~~--conflicts~~ Conflict resolved in build [OMERO-guides-push#118](https://latest-ci.openmicroscopy.org/jenkins/job/OMERO-guides-push/118/). See the [console output](https://latest-ci.openmicroscopy.org/jenkins/job/OMERO-guides-push/118/consoleText) for more details.

File: docs/conf.py [hunk 1]
```diff
@@ -50,7 +50,7 @@
 
 # General information about the project.
 project = u'OMERO.iviewer guide'
-copyright = u'2019, Open Microscopy Environment'
+copyright = u'2019-2020, Open Microscopy Environment'
```
Comment (pwalczysko) [inline docs/conf.py]:
```suggestion
copyright = u'2019-2021, Open Microscopy Environment'
```

Not sure, but just a suggestion, because it is a new year ?

**Comment by pwalczysko** [comment]
One suggestion Re: coyright year, otherwise ready to merge fmpov.

File: docs/conf.py (hunk 1 above)
Comment (jburel) [reply docs/conf.py]: PR was opened in 2020 :-)

**Review by pwalczysko — APPROVED** [review]

**Pull request merged**